GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivityBase.java

```diff
@@ -123,6 +121,7 @@ public void run() {
     abstract GVRActivityNative makeActivityNative();
     abstract GVRViewManager makeViewManager(GVRXMLParser xmlParser);
     abstract GVRMonoscopicViewManager makeMonoscopicViewManager(GVRXMLParser xmlParser);
+    abstract GoogleVRViewManager makeGoogleVRViewManager(GVRXMLParser xmlParser);
```

Comment:
Do you want to limit Google VR to only the oculus backend?

Reg line 247, we do not want to register a draw frame listener for google vr or monoscopic.

Also I noticed that the mDockEventReceiver is initialized twice in this class - line 104 and 568